src/main/java/com/global/api/entities/exceptions/GatewayTimeoutException.java

```diff
@@ -53,9 +53,9 @@ public void setTransmissionTime(String transmissionTime) {
     }
 
     public GatewayTimeoutException() {
-        super("The gateway did not respond within the given timeout.");
+        super("The gateway did not response with given timeout.");
```

Comment:
This was correct the way it was

Reply:
Nothing to do with this. I my not the one who optimize the given parameter string of that base class.